Client/sdk/game/CAnimBlendAssocGroup.h

```diff
@@ -19,6 +19,160 @@ struct RpClump;
+    ANIM_GROUP_STRIP = 51,
+    ANIM_GROUP_GANGS = 52,
+    ANIM_GROUP_ATTRACTORS = 53,
+    ANIM_GROUP_PLAYER = 54,
+    ANIM_GROUP_FAT = 55,
+    ANIM_GROUP_MUSCULAR = 56,
+    ANIM_GROUP_PLAYERROCKET = 57,
+    ANIM_GROUP_PLAYERROCKETF = 58,
+    ANIM_GROUP_PLAYERROCKETM = 59,
+    ANIM_GROUP_PLAYER2ARMED = 60,
+    ANIM_GROUP_PLAYER2ARMEDF = 61,
+    ANIM_GROUP_PLAYER2ARMEDM = 62,
+    ANIM_GROUP_PLAYERBBBAT = 63,
+    ANIM_GROUP_PLAYERBBBATF = 64,
+    ANIM_GROUP_PLAYERBBBATM = 65,
+    ANIM_GROUP_PLAYERCSAW = 66,
+    ANIM_GROUP_PLAYERCSAWF = 67,
+    ANIM_GROUP_PLAYERCSAWM = 68,
+    ANIM_GROUP_PLAYERSNEAK = 69,
+    ANIM_GROUP_PLAYERJETPACK = 70,
+    ANIM_GROUP_SWIM = 71,
+    ANIM_GROUP_DRIVEBYS = 72,
+    ANIM_GROUP_BIKE_DBZ = 73,
+    ANIM_GROUP_COP_DBZ = 74,
+    ANIM_GROUP_QUAD_DBZ = 75,
+    ANIM_GROUP_FAT_TIRED = 76,
+    ANIM_GROUP_HANDSIGNAL = 77,
+    ANIM_GROUP_HANDSIGNALL = 78,
+    ANIM_GROUP_LHAND = 79,
+    ANIM_GROUP_RHAND = 80,
+    ANIM_GROUP_CARRY = 81,
+    ANIM_GROUP_CARRY05 = 82,
+    ANIM_GROUP_CARRY105 = 83,
+    ANIM_GROUP_INT_HOUSE = 84,
+    ANIM_GROUP_INT_OFFICE = 85,
+    ANIM_GROUP_INT_SHOP = 86,
+    ANIM_GROUP_STEALTH_KN = 87,
+    ANIM_GROUP_STDCARAMIMS = 88,
+    ANIM_GROUP_LOWCARAMIMS = 89,
+    ANIM_GROUP_TRKCARANIMS = 90,
+    ANIM_GROUP_STDBIKEANIMS = 91,
+    ANIM_GROUP_SPORTBIKEANIMS = 92,
+    ANIM_GROUP_VESPABIKEANIMS = 93,
+    ANIM_GROUP_HARLEYBIKEANIMS = 94,
+    ANIM_GROUP_DIRTBIKEANIMS = 95,
+    ANIM_GROUP_WAYFBIKEANIMS = 96,
+    ANIM_GROUP_BMXBIKEANIMS = 97,
+    ANIM_GROUP_MTBBIKEANIMS = 98,
+    ANIM_GROUP_CHOPPABIKEANIMS = 99,
+    ANIM_GROUP_QUADBIKEANIMS = 100,
+    ANIM_GROUP_VANCARANIMS = 101,
+    ANIM_GROUP_RUSTPLANEANIMS = 102,
+    ANIM_GROUP_COACHCARANIMS = 103,
+    ANIM_GROUP_BUSCARANIMS = 104,
+    ANIM_GROUP_DOZERCARANIMS = 105,
+    ANIM_GROUP_KARTCARANIMS = 106,
+    ANIM_GROUP_CONVCARANIMS = 107,
+    ANIM_GROUP_MTRKCARANIMS = 108,
+    ANIM_GROUP_TRAINCARRANIMS = 109,
+    ANIM_GROUP_STDTALLCARAMIMS = 110,
+    ANIM_GROUP_HOVERCARANIMS = 111,
+    ANIM_GROUP_TANKCARANIMS = 112,
+    ANIM_GROUP_BFINJCARAMIMS = 113,
+    ANIM_GROUP_LEARPLANEANIMS = 114,
+    ANIM_GROUP_HARRPLANEANIMS = 115,
+    ANIM_GROUP_STDCARUPRIGHT = 116,
+    ANIM_GROUP_NVADAPLANEANIMS = 117,
+    ANIM_GROUP_MAN = 118,
+    ANIM_GROUP_SHUFFLE = 119,
+    ANIM_GROUP_OLDMAN = 120,
+    ANIM_GROUP_GANG1 = 121,
+    ANIM_GROUP_GANG2 = 122,
+    ANIM_GROUP_OLDFATMAN = 123,
+    ANIM_GROUP_FATMAN = 124,
+    ANIM_GROUP_JOGGER = 125,
+    ANIM_GROUP_DRUNKMAN = 126,
+    ANIM_GROUP_BLINDMAN = 127,
+    ANIM_GROUP_SWAT = 128,
+    ANIM_GROUP_WOMAN = 129,
+    ANIM_GROUP_SHOPPING = 130,
+    ANIM_GROUP_BUSYWOMAN = 131,
+    ANIM_GROUP_SEXYWOMAN = 132,
+    ANIM_GROUP_PRO = 133,
+    ANIM_GROUP_OLDWOMAN = 134,
+    ANIM_GROUP_FATWOMAN = 135,
+    ANIM_GROUP_JOGWOMAN = 136,
+    ANIM_GROUP_OLDFATWOMAN = 137,
+    ANIM_GROUP_SKATE = 138,
+    TOTAL_ANIM_GROUPS = 139
+};
+
+enum eMoveAnims
```

Comment:
are we able to use use enum class?

Reply:
Yes, that's definitely better. I forgot.